task/07-yield-tasks.js

```diff
@@ -33,7 +33,14 @@
  *
  */
 function* get99BottlesOfBeer() {
-    throw new Error('Not implemented');
+    let restBottles = 99;
+    while(restBottles >= 1){
```

Comment:
it is necessary to observe indents between operators and brackets.
https://javascript.info/coding-style